### PR TITLE
iteration 0 initial layout

### PR DESCRIPTION
**Pull request by BigBike96**
## Is this a fix or a feature?

Feature

## What is the change?

Iteration 0 html/css layout to match comp

## What does it fix?

n/a

## Where should the reviewer start?

Start on first line of html and css file(s)

## How should this be tested?

open index.html to make sure it matches comp

### CODE REVIEW

**Review by Shakikka — APPROVED** [review]
Looks very close to the given comp model.

**Review by ConnorAndersonLarson — APPROVED** [review]
Looks good and working! I think we should probably go back and refactor/standardize our CSS layouts but other than that looks great!

File: styles.css [hunk 1]
```diff
@@ -1 +1,160 @@
...
+  padding: 50px 140px 50px 140px;
+  box-sizing: border-box;
+}
+
+.button-box {
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+  width: 100%;
+  }
```
Comment (ConnorAndersonLarson) [inline styles.css:70]: remove space

File: styles.css [hunk 2]
```diff
@@ -1 +1,160 @@
...
+    align-items: center;
+    background: transparent;
+    font-size: 110%;
+    color: #CBC9CF;
+    height: 67px;
+    width: 155px;
+    border: 2px solid #FFF;
+    border-radius: 5px;
+    padding-bottom: 7px;
+  }
```
Comment (ConnorAndersonLarson) [inline styles.css:91]: remove indentation 1 level.

File: styles.css [hunk 3]
```diff
@@ -1 +1,160 @@
+html {
+  height: 100%;
+}
 
+body {
+  background-color: #2B2733;
+  display:flex;
+  flex-direction:column;
```
Comment (ConnorAndersonLarson) [inline styles.css:8]: Gotta go through and standardize (lots of things don't have spaces, CSS could be better ordered).

File: styles.css [hunk 4]
```diff
@@ -1 +1,160 @@
...
+}
+/* Main background: #2B2733
+Text on main background: #CBC9CF
+Header, cards: #46424D
+Card shadow: #26222D
+Text on header and cards: #FFF
+Study: #B3FD78
+Meditate: #C278FD
+Exercise: #FD8078
+Error: #EFB7EC */
```
Comment (ConnorAndersonLarson) [inline styles.css:160]: Not necessary now but we'll have to remove these later.

**Pull request merged**